Zend/Optimizer/zend_func_info.c

```diff
@@ -499,11 +492,6 @@ static const func_info_t old_func_infos[] = {
 	F1("iconv_mime_decode",                     MAY_BE_FALSE | MAY_BE_STRING),
 	F1("iconv_mime_decode_headers",             MAY_BE_FALSE | MAY_BE_ARRAY | MAY_BE_ARRAY_KEY_STRING | MAY_BE_ARRAY_OF_STRING | MAY_BE_ARRAY_OF_ARRAY),
 
-	/* ext/json */
-	F1("json_encode",                           MAY_BE_FALSE | MAY_BE_STRING),
-	FN("json_decode",                           MAY_BE_ANY | MAY_BE_ARRAY_KEY_ANY | MAY_BE_ARRAY_OF_ANY),
```

Comment:
am I right that this entry is unnecessary?

Reply:
It's missing MAY_BE_ARRAY_OF_REF, but I'm okay with dropping it.